data.js

```diff
@@ -0,0 +1,47 @@
+const artists = [
+  {
+    name: 'Thu Ya Kyaw',
+    position: 'Developer Advocate for Google Cloud (AI/ML)',
+    description:
+      'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis illum animi iure numquam. Rem sunt ipsam, omnis voluptate quia minus deleniti dolor, recusandae nihil adipisci placeat fuga nesciunt sed quibusdam.orem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet con',
+    image: 'ThuYaKyaw.jpg',
+  },
+  {
+    name: 'Si Thu Maung',
+    position: 'People Strategist | Edu-Tech Entrepreneur',
+    description:
+      'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis illum animi iure numquam. Rem sunt ipsam, omnis voluptate quia minus deleniti dolor, recusandae nihil adipisci placeat fuga nesciunt sed quibusdam.orem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet con',
+    image: 'SiThuMg.jpeg',
+  },
+  {
+    name: 'Khine Zar Thwe',
+    position: 'Application Engineer',
+    description:
+      'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis illum animi iure numquam. Rem sunt ipsam, omnis voluptate quia minus deleniti dolor, recusandae nihil adipisci placeat fuga nesciunt sed quibusdam.orem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet con',
+    image: 'KhineZarThwe.jpg',
+  },
+  {
+    name: 'Su Myat Noe',
+    position: 'Women in AI Myanmar Ambassador',
+    description:
+      'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis illum animi iure numquam. Rem sunt ipsam, omnis voluptate quia minus deleniti dolor, recusandae nihil adipisci placeat fuga nesciunt sed quibusdam.orem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet con',
+    image: 'SuMyatNoe.png',
+  },
+];
+
+const artistMainCard = document.getElementById('artist-data');
+artists.map((a) => {
+  artistMainCard.innerHTML += `
+    <div class="artist-card">
+                <img src="./images/${a.image}" alt="" class="artist-image">
+                <div class="artist-description">
+                    <h3 class="artist-name">${a.name}</h3>
+                    <h4 class="artist-title">${a.position}</h4>
+                    <p class="artist-breif">
+                        ${a.description}
+                    </p>
+                </div>
+            </div>
+`;
+  return true;
+});
```

Comment:
- [ ] Good job :+1:  but in this code, you are also not using DRY principles as I mentioned above :point_up: please fix duplication :slightly_smiling_face:  and also check my suggestion below :point_down: 

```suggestion
const artists = [
  {
    name: 'Thu Ya Kyaw',
    position: 'Developer Advocate for Google Cloud (AI/ML)',
    description:
      'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis illum animi iure numquam. Rem sunt ipsam, omnis voluptate quia minus deleniti dolor, recusandae nihil adipisci placeat fuga nesciunt sed quibusdam.orem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet con',
    image: 'ThuYaKyaw.jpg',
  },
  {
    name: 'Si Thu Maung',
    position: 'People Strategist | Edu-Tech Entrepreneur',
    description:
      'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis illum animi iure numquam. Rem sunt ipsam, omnis voluptate quia minus deleniti dolor, recusandae nihil adipisci placeat fuga nesciunt sed quibusdam.orem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet con',
    image: 'SiThuMg.jpeg',
  },
  {
    name: 'Khine Zar Thwe',
    position: 'Application Engineer',
    description:
      'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis illum animi iure numquam. Rem sunt ipsam, omnis voluptate quia minus deleniti dolor, recusandae nihil adipisci placeat fuga nesciunt sed quibusdam.orem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet con',
    image: 'KhineZarThwe.jpg',
  },
  {
    name: 'Su Myat Noe',
    position: 'Women in AI Myanmar Ambassador',
    description:
      'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Nobis illum animi iure numquam. Rem sunt ipsam, omnis voluptate quia minus deleniti dolor, recusandae nihil adipisci placeat fuga nesciunt sed quibusdam.orem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet conorem ipsum, dolor sit amet con',
    image: 'SuMyatNoe.png',
  },
];

const generateArtistCard = ({ name, position, description, image }) => `
  <div class="artist-card">
    <img src="./images/${image}" alt="" class="artist-image">
    <div class="artist-description">
      <h3 class="artist-name">${name}</h3>
      <h4 class="artist-title">${position}</h4>
      <p class="artist-breif">
        ${description}
      </p>
    </div>
  </div>
`;

const artistMainCard = document.getElementById('artist-data');
artists.map(generateArtistCard).forEach(card => artistMainCard.innerHTML += card);
```
**The main changes are:**

- Creating a separate function to generate the HTML for each artist card, and using destructuring to simplify the code.
- Using `forEach` instead of `map` to avoid returning an unnecessary array.
- Calling the `generateArtistCard` function for each artist in the array using `forEach` and adding the resulting HTML to the `artistMainCard` element.